packages/@lwc/engine-core/src/framework/update-component-value.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { VM } from './vm';
+import { componentValueMutated } from './mutation-tracker';
+
+export function updateComponentValue(vm: VM, key: string, newValue: any) {
```

Comment:
We do have a lot of files under the `framework` directory. Could this standalone method be grouped with some other logic in a different file?

Reply:
@pmdartus I think it's a personal preference thing, but I'm a big fan of "lots of different small files" rather than `utils.js` (or similar). My reasoning is:

1. It's clear that none of the utils rely on each other
2. For subtle reasons, bundlers may fail to tree-shake code if it's co-located in the same file
3. Files are free

But I admit this is just my personal preference mostly.

Reply:
I don't have a strong opinion on this.